packages/compass/src/main/auto-update-manager.ts

```diff
@@ -618,26 +631,37 @@ class CompassAutoUpdateManager {
     return url;
   }
 
-  static async checkForUpdate(): Promise<{
-    name: string;
-    from: string;
-    to: string;
-  } | null> {
+  static async checkForUpdate(): Promise<AutoUpdateResponse> {
     try {
       const response = await this.fetch((await this.getUpdateCheckURL()).href);
-      if (response.status !== 200) {
-        return null;
-      }
+
+
+      assert(response.ok);
```

Comment:
Are we changing the backend to return more success status codes than 200? I don't think we expect anything else but 200 as a success state, not sure why we're making the check less strict here

Reply:
I can revert change and return early instead 👍